lib/qme/map/map_reduce_builder.rb

```diff
@@ -32,6 +32,9 @@ def self.add_defaults(vars)
           if !vars.has_key?('enable_rationale')
             vars['enable_rationale'] = false
           end
+          if !vars.has_key?('short_circuit')
```

Comment:
This does indeed seem to be the culprit .... 

That corresponds with the intel gathered RE: the `Unexpected token ;` error showing up when missing default parameters per that email you shared earlier.